iterate/multiple-vectors/multiple-vectors.Rmd

```diff
@@ -349,11 +349,9 @@ Although its name does not suggest it, `pmap()` is an important tool for manipul
 
 Recall from the Introduction to Iteration tutorial that a data frame is a list of column vectors that contains a special class ("data.frame") and a row.names attribute.
 
-Because a data frame is a list, you can pass a data frame to the first argument of `pmap()`. `pmap()` will do something natural when you do this...
-
-### rowwise operations
-
-`pmap()` applies a function to each row of a data frame.
+Because a data frame is a list, you can pass a data frame to the first argument of `pmap()`. `pmap()` will do something natural when you do this:
```

Comment:
Can you restore a section break somewhere in here? The section breaks control the advance of material, and this ends up being a lot of material to present all at once.

Reply:
Restored.